.github/workflows/cd.yml

```diff
@@ -16,15 +19,21 @@ jobs:
       contents: write
 
     steps:
-      - uses: actions/checkout@v3
-      - uses: softprops/action-gh-release@v1
+      - uses: actions/checkout@v4
+      - name: "Version Release"
```

Comment:
I'd omit the name here as the action name is self-descriptive. Thoughts?

Reply:
Name is used in actions [view](https://github.com/nikitastupin/clairvoyance/actions/runs/8900538585/job/24442321031), I guess that maybe name could be more descriptive here (since it's more like posting artifacts). 

And as for me the name should be viewed as a step name, rather than as a comment on step. Like a function name in programming. You can see [here](https://docs.github.com/en/actions/examples/using-scripts-to-test-your-code-on-a-runner) that even "checkout" has its own name in the examples provided by Github.

I guess "Version release on Github" would be more fitting.